### PR TITLE
Activate stale bot for pull requests

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

The stale bot has worked well for issues since activation.
Activating this for pull requests will allow developers to focus
their efforts on reviewing relevant work.

**To test:**

* Review the changes, particularly the comments applied to the pull requests when the bot runs

*There is no associated issue.*

*This does not require release notes* because **it does not alter the code base and is a process change**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
+    activity in 3 months. It will be closed in 7 days if no further activity occurs.
+
+    Allowing pull requests to close as stale helps us filter out old work that is no longer
+    relevant and helps developers focus on reviewing current work.
+
+    All pull requests closed by this bot act like normal pull requests; they can be searched for, commented on or reopened at any point.
+
+    This pull request has been closed automatically. If these changes are still relevant
+    then please re-open this pull request with a comment and tag \@mantidproject/gatekeepers
+    to highlight it needs to have a reviewer assigned.
```
Comment (DavidFair) [inline .github/stale.yml:58]: I think this should be in the next section or removed? At this point the PR is still in the "warning" phase

**Review by DavidFair — CHANGES_REQUESTED** [review]

File: .github/stale.yml (hunk 1 above)
Comment (martyngigg) [reply .github/stale.yml:58]: Oh yes good spot. Copy-and-paste error! I think what I meant to keep was just
*"If these changes are still relevant
    then please comment and tag \@mantidproject/gatekeepers
    to highlight it needs to have a reviewer assigned."*

Is that worth keeping do you think?

Comment (DavidFair) [reply .github/stale.yml:58]: I think so, if people start abusing it for everything we can always remove that note too

Comment (martyngigg) [reply .github/stale.yml]: Thanks David. This should be corrected now.

**Review by DavidFair — APPROVED** [review]
LGTM
The only concern I have is if the bot's message unintentionally pings gatekeepers when it posts the messages

But the best way to find out is by giving it a spin

**Comment by martyngigg** [comment]
> LGTM The only concern I have is if the bot's message unintentionally pings gatekeepers when it posts the messages

Yeah I was concerned about that too. I put a backslash in front of the `@` and I *think* this should mean it displays without being a reference that causes a ping. Let's see!

**Comment by martyngigg** [comment]
@mantidproject/gatekeepers Could I get a merge on this?

**Pull request merged**